src/components/ProvidersSummary/ProvidersSummary.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { string, number } from 'prop-types';
+import {
+  StyleSheet, Text, View,
+} from 'react-native';
+
+import Colors from '../../constants/Colors';
+import { clearPatientData } from '../../features/patient/patientDataSlice';
+
+const ProviderRow = ({ name, latestDataYear }) => (
+  <View style={styles.providerTypeRow}>
+    <Text style={styles.providerName}>{name}</Text>
+    <Text style={styles.providerLatestDate}>{latestDataYear}</Text>
+  </View>
+);
+
+ProviderRow.propTypes = {
+  name: string.isRequired,
+  latestDataYear: number.isRequired,
+};
+
+const ProvidersSummary = () => {
+  const providers = [
+    { name: 'LAC/Olive View-UCLA Medical Center', latestDataYear: 2020 },
+    { name: 'Pacific Alliance Medical Center', latestDataYear: 2018 },
+  ];
```

Comment:
These shouldn't be placesholder... But we'll need to do a follow up data request to get `Providers` from `resources`. We may have to punt on that until later.

Reply:
Agree on the placeholder - but there's no `Providers` in FHIR. The desktop app clearly uses two different 'EHRs' for retrieving data - I assume the same is planned for mobile?